packages/app-support/limber-ui/addon/package.json

```diff
@@ -38,7 +38,8 @@
     "ember-modifier": "^4.1.0",
     "ember-primitives": ">= 0.11.0",
     "ember-resources": ">= 6.4.2",
-    "ember-source": ">= 5.9.0"
+    "ember-source": ">= 5.10.0",
+    "webpack": "^5.92.1"
```

Comment:
adding webpack as a peer here to "forward" ember-source's peer resolves the peer issues embroider is telling us about (duplicate ember-sources)

Reply:
this feels super gross though, and I've asked about it here https://discord.com/channels/480462759797063690/568935504288940056/1260033731918299136